### ISSUE TITLE
loss is still 10.3750

### ISSUE THREAD

**Issue by Rh-Dang** [Vision-CAIR/MiniGPT-4#267]
No matter what dataset and what pre-training checkpoint, the loss is always maintained at about 10.3750 during the second stage train. Do you have this question? How can I solve?

**Comment by hangzeli08** [comment]
I meet the same question ,could you tell me how do you solve it?
